### ISSUE TITLE
State Information Should React With Brevity Not Truncation in WebUI

### ISSUE THREAD

**Issue by markmcb** [syncthing/syncthing#5703]
### What happened

When viewing a system with a large number of files, folders, and total storage, in a narrow browser some of the "Local State (Total)" information is truncated and only visible with a mouse over.

### What you expected to happen instead

Use shorter, less specific data on-screen instead of omitting data. Example:

For "This Device" I see (screen shot attached):
[file icon] 491,796 [folder icon] 24,614 [disk icon] ...                

<img width="488" alt="Screen Shot 2019-05-09 at 9 49 02 AM" src="https://user-images.githubusercontent.com/54813/57472655-da74b380-7242-11e9-8a23-13805747c0b9.png">

I'd rather see two significant digits and a unit:
[file icon] 0.4M [folder icon] 24K [disk icon] 9.9TiB

![Screen Shot 2019-05-09 at 10 16 37 AM](https://user-images.githubusercontent.com/54813/57472964-9f26b480-7243-11e9-8699-480c29d67514.png)

... or something like that, and a mouse over would show the exact data.

# Any steps to reproduce the problem

View a system with a large number of files and folders.

### Version Information

Syncthing Version: v1.1.3
OS Version: Fedora 30
Browser Version: Firefox 66

**Comment by AudriusButkevicius** [comment]
No, because a difference of 1 item can be all it takes to undersrand why it's out of sync, etc.
These specified fields being detailed helped in hundreds of debugging cases.

**Comment by markmcb** [comment]
This is for the reactive aspect. I'm not suggesting you get rid of the information. Both a wider browser and mouse over would provide the information.

If you're certain it should all be displayed all the time, then I'd add another line so that you avoid truncation, i.e., apply your rationale to the total storage value.

From a design standpoint, you already have an inconsistency, i.e., you show exact files, exact folders, and approximate storage. I'd keep each line consistent. If it's exact data for debugging, it should all be exact.

**Comment by AudriusButkevicius** [comment]
Sure, that's a fair ask in the reactive scenario, and I guess we could compensate with alt text.

**Comment by wweich** [comment]
We already go "multi line" for small screens (e.g. 450px wide):
![grafik](https://user-images.githubusercontent.com/1026763/57478330-e8eeb980-729a-11e9-8cbe-e2e3edd5ea07.png)

But not for mid sized ones (e.g. 1000px wide / 2 columns):
![grafik](https://user-images.githubusercontent.com/1026763/57478374-03289780-729b-11e9-92d6-47b0a01a5b2f.png)

Should we not just do it all the time? _And set a nobr or something, so the icon stays on the same line as the number._

**Comment by calmh** [comment]
Yes, we should. I don't know why we attempt to make that distinction. I remember we fixed something similar about this layout not too far back, too.
